frontend/src/app/services/auth/auth.service.ts

```diff
@@ -38,6 +38,10 @@ export class AuthService {
     }
```

Comment:
In this service i missed previous time 1 point. What should do method checkToken. I don't understand, what does it check? As i understodd whis method should check on expire, but why in this case method which make check runs logout?